geocode_lat_long/geocode_lat_long.md

```diff
@@ -0,0 +1,263 @@
+<!--
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  1.0.0
+module_template_version: 3.0.0
+language: en
+narrator: UK English Female
+title: Encoding Geospatial Data: Latitude and Longitude
+comment:  This is an introduction to latitude and longitude and the importance of geocoding - encoding geospatial data in the coordinate system.
+long_description: If you use any geospatial data, such as patient or participant addresses, it is important that that location data be in a usable form. This means using the same coordinate system that Global Positioning Systems use: latitude and longitude. This module is appropriate, as either an introduction or review, for anyone considering using geospatial data in their analysis. 
+estimated_time: 15 minutes
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Understand the importance of geocoding addresses
+- Understand the latitude and longitude coordinate system
+- Geocode single addresses.  
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Encoding Geospaital Data: Latitude and Longitude
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites**
+None
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## Where is it Located?
+
+"Where is the research building?" If you have ever need to go somewhere new, you have run into the primary issue with the way humans discuss location.
+
+Depending on who (or what) you ask, the answer could take many forms, each of which can be useful in some situations and useless in others. Consider the types of answers you could get to the question "Where is the Roberts Center for Pediatric Research?"
+
+- **Relational Location**: "The research building is on the other side of the South St. bridge from the hospital." 
+
+- **Directions**: "Go three blocks that way, turn left, and you will see the research building." 
+
+- **Street Address**: "The research building is at 734 Schuylkill Ave."
+
+- **Mailing Address**: "The research building is at 734 Schuylkill Ave. Philadelphia, PA 19146."
+
+- **Coordinates**: "The research building is at 39.9458 North, -75.1865 West."
+
```

Comment:
<div class = "care">
<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>

Place names add an additional level of difficulty to working with geospatial data. People may notice and care if you use the wrong name for a place, or if you mispronounce a common local name. When in doubt about how to pronounce a place name, it is always a good idea to ask!

While you won't need to speak any place names out loud for this module, Philadelphians pronounce Schuylkill as "SKOO-kill."

</div>